code/game/objects/items/storage/firstaid.dm

```diff
@@ -363,11 +363,11 @@
 	icon_state = "pill_canister11"
 	pill_type_to_fill = /obj/item/reagent_containers/pill/bicaridine
 
-/obj/item/storage/pill_bottle/bicamera
-	name = "bicamera pill bottle"
+/obj/item/storage/pill_bottle/meralyne
+	name = "meralyne pill bottle"
 	desc = "Contains pills that heal cuts and bruises, but cause slight pain. Take two to heal faster, but have slightly more pain."
 	icon_state = "pill_canister11"
-	pill_type_to_fill = /obj/item/reagent_containers/pill/bicamera
```

Comment:
but why

Reply:
HANGES-

Multi-reagent pills should not exist roundstart, period. That's what autoinjectors and the chemmaster's for. Put some slight effort into your one-click medic build.
Removed the bicard and kelotane, from bicamera and keloderm pills left the dermaline and meralyne. Amounts NOT increased for medical vendor prici

Reply:
> `Multi-reagent pills should not exist roundstart, period. That's what autoinjectors and the chemmaster's for. Put some slight effort into your one-click medic build.`
> 
> This enhances the experience how exactly?
> 
> There is a single grinder/chem station meaning I will have to wait for everyone else, and people recently have recently expressed displeasure at long prep times, notably which are brought around by the pillars of medic prep, engi prep and req prep and that change in no way helps this.

does not answer the question